translations/strings.yaml

```diff
@@ -63,3 +63,22 @@ settingsDev:
   useStagingTitle: Use Staging Servers
   useStagingSubtitle: Staging services are used to test features before they go live to production
   restartRequired: VPN must be restarted to take effect
+
+splittunnel:
+  missingAppAction:
+    value:
+      - Add apps manually
+    comment:
+      - Refers to the Apps that are Missing from the Split Tunnel list
+  missingAppOne: 
+    value:
+      - \%0 Apps missing from list
```

Comment:
"App" -> "app" ? Check with @flodolo the right string to use here.

Reply:
Personally I think that _apps_ should be lowercase. But that's more a question for who owns UX/content than l10n (not sure that was the question though).

With that said:
* Why the escaping? How does the string looks like once exported to .ts? 
* There should always be a comment explaining what replaces`%0`, as the linter correctly points out for `vpn.splittunnel.missingAppAddedOne`.

Having 2 strings for singular/plural is not helpful: we conveniently make it work for English and another bunch of languages, but mess with the others. We should try to find a string that works for all locales, e.g. `Apps missing from list: %0`. For the record, that's what Google suggests when it comes to plurals in WebExtensions ([reference](https://www.chromium.org/developers/design-documents/extensions/how-the-extension-system-works/i18n#TOC-Plural-form)).

Reply:
Good point. Let's remove the singular string. 

A line beginning with % is reserved for YAML directives, that's why it's escaped 
the exported .ts does not contain it e.g
```
<message id="vpn.splittunnel.missingAppMultiple">
        <source>%0 apps missing from list.</source>
        <oldsource>%0 Apps missing from list.</oldsource>
        <extracomment>apps in the Split-Tunnel settings. %0 is the amount of apps missing</extracomment>
        <translation type="unfinished"></translation>
    </message>
````